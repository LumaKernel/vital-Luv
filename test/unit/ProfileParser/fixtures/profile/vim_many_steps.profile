SCRIPT  /home/luma/vim-make-plugin/vital-Luv/dev/ProfileParser/test_many_steps.vim
Sourced 1 time
Total time:   4.426330
 Self time:   4.426330

count  total (s)   self (s)
    1              0.000007 let x = 0
1000001              0.996196 for i in range(1000000)  " 1e6
1000000              1.130270   let x += 1
1000001              0.801945 endfor
    1              0.000013 echo x
                            

