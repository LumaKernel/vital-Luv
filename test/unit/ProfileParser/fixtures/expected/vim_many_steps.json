[{"path":"/home/luma/vim-make-plugin/vital-Luv/dev/ProfileParser/test_many_steps.vim","total_time":4.42633,"self_time":4.42633,"type":"script","lines":[{"self_time":7.0e-6,"count":1,"content":"let x = 0"},{"self_time":0.996196,"count":1000001,"content":"for i in range(1000000)  \" 1e6"},{"self_time":1.13027,"count":1000000,"content":"  let x += 1"},{"self_time":0.801945,"count":1000001,"content":"endfor"},{"self_time":1.3e-5,"count":1,"content":"echo x"},{"content":""}],"count":1}]
